src/commands/account/storage_management/mod.rs

```diff
@@ -0,0 +1,64 @@
+use strum::{EnumDiscriminants, EnumIter, EnumMessage};
+
+mod storage_deposit;
+mod storage_withdraw;
+mod view_storage_balance;
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(input_context = crate::GlobalContext)]
+#[interactive_clap(output_context = ContractContext)]
+pub struct Contract {
+    /// Which contract account ID do you want to view the balance?
+    contract_account_id: crate::types::account_id::AccountId,
+    #[interactive_clap(subcommand)]
+    storage_actions: StorageActions,
+}
+
+#[derive(Clone)]
+pub struct ContractContext {
+    pub config: crate::config::Config,
+    pub get_contract_account_id: GetContractAccountID,
+}
+
+impl ContractContext {
+    pub fn from_previous_context(
+        previous_context: crate::GlobalContext,
+        scope: &<Contract as interactive_clap::ToInteractiveClapContextScope>::InteractiveClapContextScope,
+    ) -> color_eyre::eyre::Result<Self> {
+        let contract_account_id = scope.contract_account_id.clone();
+        let get_contract_account_id: GetContractAccountID =
+            std::sync::Arc::new(move |_network_config| Ok(contract_account_id.clone().into()));
+        Ok(Self {
+            config: previous_context.0,
+            get_contract_account_id,
+        })
+    }
+}
+
+pub type GetContractAccountID = std::sync::Arc<
```

Comment:
We use `Id` (with lower-case `d`) in other places, so let's rename it for consistency:

```suggestion
pub type GetContractAccountId = std::sync::Arc<
```